packages/material-ui/src/OutlinedInput/NotchedOutline.js

```diff
@@ -96,7 +98,14 @@ const NotchedOutline = React.forwardRef(function NotchedOutline(props, ref) {
         >
           {/* Use the nominal use case of the legend, avoid rendering artefacts. */}
           {/* eslint-disable-next-line react/no-danger */}
```

Comment:
This needs to be moved down right above the `dangerouslySetInnerHTML`

Reply:
Thank you! Didn't even see that one, been a long day at work. 
Not sure where to place the "Use the nominal use case" comment tho, as I'm not really sure what it's referring to.